src/Http/Controllers/WebhookController.php

```diff
@@ -222,4 +222,26 @@ protected function findOrCreateCustomer(string $passthrough)
             'billable_type' => $passthrough['billable_type'],
         ])->billable;
     }
+
+    /**
+     * Find the first subscription matching a Paddle subscription id.
+     *
+     * @param  string  $subscriptionId
+     * @return \Laravel\Paddle\Subscription|null
+     */
+    protected function findSubscription(string $subscriptionId)
+    {
+        return Subscription::firstWhere('paddle_id', $subscriptionId);
+    }
+
+    /**
+     * Determine if a receipt with a given Order ID already exists.
+     *
+     * @param  string  $orderId
+     * @return bool
+     */
+    protected function doesReceiptExists(string $orderId)
```

Comment:
Either `doesReceiptExist` or `receiptExists` I think would be best here?

Reply:
Fixed